presto-main/src/test/java/com/facebook/presto/type/TestCharOperators.java

```diff
@@ -132,4 +132,17 @@ public void testBetween()
         assertFunction("cast('bar' as char(4)) BETWEEN cast('bar' as char(3)) AND cast('bar' as char(5))", BOOLEAN, true);
         assertFunction("cast('bar' as char(4)) BETWEEN cast('bar' as char(5)) AND cast('bar' as char(7))", BOOLEAN, false);
     }
+
+    @Test
+    public void testIsDistinctFrom()
+            throws Exception
+    {
+        assertFunction("cast(NULL as char(3)) IS DISTINCT FROM cast(NULL as char(3))", BOOLEAN, false);
```

Comment:
Add `cast(NULL as char(3)) IS DISTINCT FROM cast(NULL as char(5))`. I believe the expected value is `false`.